Loop/Managers/Alerts/InAppModalAlertIssuer.swift

```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2020 LoopKit Authors. All rights reserved.
 //
 
-import Foundation
+import UIKit
```

Comment:
[nit] I'm not sure I see why this change was necessary (?)

Reply:
Same answer for this and the other comments below: LoopKit was including UIKit in its umbrella header, which I removed. These files use various symbols from UIKit, and so it needs to be imported explicitly.